### PR TITLE
Fix for(let x of y.prop)

### LINKED ISSUE #6: for(let x of y.prop) being incorrectly replaced with for(let t of t.prop)
This bug came about via the Visual Studio Bundler and Minifier extension, but is apparently a bug in this library:

**Description:**

Variables in for (x of y.property) loop are being replaced with for(let t of t.prop) and causing problems when t gets overwritten.
Steps to recreate

Minify the following snippet:

```
function successCallback(response) {
   if (response.data) {
      for (let thing of response.data) {
         if (!someArray.find(function (item) { return item.prop == thing.prop; })) {
            someArray.push(thing);
         }
      }
   }
}
```

**Current behavior**

Minifies to:
`function successCallback(n){if(n.data)for(let n of n.data)someArray.find(function(t){return t.prop==n.prop})||someArray.push(n)}`

let n of n.data is incorrect. The "let n" should be replaced by another letter, as n effectively overwrites the variable being referred to by "response" in the original code, breaking the reference.

If the for loop contains a direct reference to response.data (as opposed to the reference inside the anonymous inner function) the code gets minified correctly.

**Expected behavior**

Adding in references to the looped objects (console.logged) gives correct output:

```
function successCallback(response) {
   if (response.data) {
      for (let thing of response.data) {
         console.log(thing);
         console.log(response.data);
         if (!someArray.find(function (item) { return item.prop == thing.prop; })) {
            someArray.push(thing);
         }
      }
   }
}
```

`function successCallback(n){if(n.data)for(let t of n.data)console.log(t),console.log(n.data),someArray.find(function(n){return n.prop==t.prop})||someArray.push(t)}`

### PR DESCRIPTION
**Pull request by Almeonamy**
Fix #6

### CODE REVIEW

File: src/NUglify.Tests/NUglify.Tests.nuget.props [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (xoofx) [inline src/NUglify.Tests/NUglify.Tests.nuget.props]: Can you remove this file from the PR?

Comment (Almeonamy) [reply src/NUglify.Tests/NUglify.Tests.nuget.props]: Done

**Comment by xoofx** [comment]
Thanks!

**Comment by Almeonamy** [comment]
Can you create new version in NuGet?

**Comment by xoofx** [comment]
yep just pushed to 1.5.13,

**Pull request merged**